packages/blade/src/components/BottomSheet/BottomSheet.native.tsx

```diff
@@ -274,17 +277,19 @@ const _BottomSheet = ({
             enableOverDrag
             enableContentPanningGesture
             ref={sheetRef}
-            index={-1}
+            // on initial render if _isOpen is true we want to render the sheet at initialSnapPoint
+            // otherwise we want to render it at -1 so that it is not visible
+            index={_isOpen ? initialSnapPoint.current : -1}
             containerStyle={{ zIndex }}
-            animateOnMount={false}
+            animateOnMount={true}
```

Comment:
Expected to be true?

Reply:
Yes, otherwise the initial mount looks very odd.